### ISSUE TITLE
File uploads using Spring Boot WebFlux

### ISSUE THREAD

**Issue by JoakimLofgren** [graphql-java-kickstart/graphql-spring-boot#637]
We are in the process of porting from servlet to WebFlux and there doesn't seem to be a way to handle file uploads in WebFlux.

This probably involves porting [GraphQLMultipartInvocationInputParser](https://github.com/graphql-java-kickstart/graphql-java-servlet/blob/master/graphql-java-servlet/src/main/java/graphql/kickstart/servlet/GraphQLMultipartInvocationInputParser.java) to WebFlux.

And creating a WebFlux [ApolloScalars](https://github.com/graphql-java-kickstart/graphql-java-servlet/blob/master/graphql-java-servlet/src/main/java/graphql/kickstart/servlet/apollo/ApolloScalars.java) with `org.springframework.http.codec.multipart.Part` instead of `javax.servlet.http.Part`.

**Comment by oliemansm** [comment]
Yes. Webflux is a bit behind in functionality compared to the servlet version. We'll have to make an overview of all functionalities missing in webflux and make sure that those get aligned.

**Comment by jhz7** [comment]
Some advance on this topic?
